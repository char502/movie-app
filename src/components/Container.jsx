import styled from 'styled-components';

export const Container = styled.div`
    max-width: 1800px;
    width: 100vw;
    padding: 0 ${props => props.theme.sizes.xLarge};
    box-sizing: border-box;
`

// Got a single search result?
// No reason to make the rest of the page look empty...
export const SearchPageContainer = styled(Container)`
    min-height: 90vh;
`

export const MediaPageContainer = styled(Container)`
    width: 100%;
    height: 100vh;
    overflow-x: hidden;
    display: flex;
    justify-content: center;
    align-items: center;
`