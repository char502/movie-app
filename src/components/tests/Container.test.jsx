import React from "react";
import {shallow} from 'enzyme';
import { Container, SearchPageContainer, MediaPageContainer } from '../Container';
import toJson from 'enzyme-to-json';

it('Button Snapshots', () => {
    expect(toJson(shallow(<Container />))).toMatchSnapshot();
    expect(toJson(shallow(<SearchPageContainer />))).toMatchSnapshot();
    expect(toJson(shallow(<MediaPageContainer />))).toMatchSnapshot();
});