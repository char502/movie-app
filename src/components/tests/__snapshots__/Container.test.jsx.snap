// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Snapshots 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
    max-width: 1800px;
    x-overflow: hidden;
    padding: 0 ",
          [Function],
          ";
    box-sizing: border-box;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Button Snapshots 2`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bwzfXH",
        "isStatic": false,
        "rules": Array [
          "
    max-width: 1800px;
    x-overflow: hidden;
    padding: 0 ",
          [Function],
          ";
    box-sizing: border-box;
",
          "
    min-height: 90vh;
",
        ],
      },
      "displayName": "Styled(styled.div)",
      "foldedComponentIds": Array [
        "sc-bdVaJa",
      ],
      "render": [Function],
      "styledComponentId": "sc-bwzfXH",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Button Snapshots 3`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-htpNat",
        "isStatic": false,
        "rules": Array [
          "
    max-width: 1800px;
    x-overflow: hidden;
    padding: 0 ",
          [Function],
          ";
    box-sizing: border-box;
",
          "
    width: 100%;
    overflow-y: hidden;
    display: flex;
    justify-content: center;
    align-items: center;
",
        ],
      },
      "displayName": "Styled(styled.div)",
      "foldedComponentIds": Array [
        "sc-bdVaJa",
      ],
      "render": [Function],
      "styledComponentId": "sc-htpNat",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
